ql/src/test/results/clientpositive/llap/explainuser_1.q.out

```diff
@@ -1331,9 +1331,9 @@ Stage-0
     Stage-1
       Reducer 3 llap
       File Output Operator [FS_16]
-        Select Operator [SEL_15] (rows=24 width=100)
+        Select Operator [SEL_15] (rows=21 width=100)
```

Comment:
I guess the row count are just estimations, not the real row count of the executed query?